uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-calendar-month.component.js

```diff
@@ -74,7 +74,8 @@ define([
      */
     function buildMonthStructure (dateMoment) {
       return {
-        index: dateMoment.month(),
+        index: dateMoment.year() + '-' + dateMoment.month(),
```

Comment:
```js
index: getMonthIndex(dateMoment),
```

Reply:
This function does not exists in the component. It exists in the controller.